LondonBritishLibrary/orient/BLorient555.xml

```diff
@@ -0,0 +1,274 @@
+  <textLang mainLang="gez"></textLang>
+  </msItem>
+
+  <msItem xml:id="ms_i1.19">
+  <locus from="105r"/>
+  <title type="complete" xml:lang="en">Various prayers, beginning with <foreign xml:lang="gez">ረስየነ፡ ድልዋነ፡ ንክሥት፡ አፉነ፡</foreign></title>
+  <textLang mainLang="gez"></textLang>
+  </msItem>
+
+  <msItem xml:id="ms_i1.20">
+  <locus from="113v"/>
+  <title type="complete" xml:lang="gez" ref="LIT2252salota">ጸሎተ፡ ዕጣን፡</title>
+  <textLang mainLang="gez"></textLang>
+  </msItem>
+
+    </msItem>
+  </msContents>
+  <physDesc>
+    <objectDesc form="Codex">
+        <supportDesc>
+            <support>
+                <material key="parchment"/>
+            </support>
+            <extent>
+                <measure unit="leaf">129</measure>
+                <measure unit="blank">1</measure>
+                <dimensions type="outer" unit="in">
+                    <height>10.375</height>
+                    <width>8.5</width>
+                </dimensions>
+                </extent>
+                <foliation>One unnumbered blank leaf at the end.</foliation>
+          </supportDesc>
+
+        <layoutDesc>
+            <layout columns="2" writtenLines="24"/>
+        </layoutDesc>
+    </objectDesc>
+    <handDesc>
+      <handNote xml:id="h1" script="Ethiopic">
+      <desc>Written in a good hand; apparently of the latter part of the <date notBefore="1750" notAfter="1800">18th Cent.</date></desc>
+      </handNote>
+      </handDesc>
+
+      <additions>
+      <list>
+
+    <item xml:id="a1">
+        <locus target="#3r"/>
+        <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+        </desc>
+        <q xml:lang="gez">ግንዘት፡ ዘቅዱስ፡ መድኀኔ፡ ዓለም።</q>
+    </item>
+
+    <item xml:id="e1">
+    <locus target="#32r #121r"/>
+    <desc>The name of the reigning king <persName ref="PRS9142TaklaGi">Takla Giyorgis</persName> is mentioned on <locus target="#32r">f.32r</locus> and f. <locus target="#121r">121r</locus>.</desc>
+    </item>
+
+    <item xml:id="e2">
+    <locus target="#8v #9v"/>
+    <desc>The names of the owners, <persName ref="PRS13287SenaGabreel">Sǝna Gabrǝʾel</persName> and <persName ref="PRS13288WalattaRufael">Walatta Fānuʾel</persName>, e. g. on <locus target="#8v">f. 8v</locus> and <locus target="#9v">f. 9v</locus>.</desc>
+    </item>
+
+
+    </list>
+  </additions>
+
+      <bindingDesc>
+          <binding contemporary="true" xml:id="binding">
+              <decoNote xml:id="b1">In the original boards, lined with flowered silk and covered with blind-tooled leather.</decoNote>
+              <decoNote xml:id="b2" type="bindingMaterial">
+                  <material key="wood"/>
+                  <material key="leather"/>
+                  <material key="silk"/>
+              </decoNote>
+          </binding>
+      </bindingDesc>
+
+
+    </physDesc>
+    <history>
```

Comment:
I think you could also add the originDate based on the dates of the reign of Takla Giyorgis. The information is implicitly provided by the catalogue, but you could turn it into a statement.